cherrypy/process/servers.py

```diff
@@ -121,6 +121,14 @@ def index(self):
 import sys
 import time
 import warnings
+import contextlib
+
+import portend
+
+
+class Timeouts:
```

Comment:
Well, I vote for having just two constants here. It's KISS. I really like this talk https://www.youtube.com/watch?v=o9pEzgHorH0, I think it's applicable, here. Two constants, which are not going to be imported somewhere externally don't need another container around them, just module itself.